lib/proxy/pluggable.rb

```diff
@@ -33,37 +33,30 @@ def after_activation
       instance_eval(&self.class.after_activation_blk) if self.class.after_activation_blk
     end
 
-    def validate!
-      validate_dependencies!(self.class.dependencies)
-      validate_prerequisites_enabled!(self.class.initialize_after)
-      execute_validators(self.class.plugin_default_settings.keys.map {|k| ::Proxy::PluginValidators::Presence.new(self.class, k)})
-      execute_validators(self.class.validators)
-    end
-
     def execute_validators(validators)
       validators.each { |validator| validator.validate! }
     end
 
-    def validate_prerequisites_enabled!(prerequisites)
-      prerequisites.each do |p|
-        if !(::Proxy::Plugins.find_plugin(p))
-          raise ::Proxy::PluginMisconfigured, "Unable to find dependency '#{p}' of '#{plugin_name}'."
-        end
-        if !(::Proxy::Plugins.plugin_enabled?(p))
-          raise ::Proxy::PluginMisconfigured, "Dependency '#{p}' of '#{plugin_name}' has not been enabled."
-        end
-      end
+    def validate!(all_plugins)
```

Comment:
Nitpick, this has moved making the diff larger than it should be, it used to be above `execute_validators`.